float32 pedestal_angle
float32 arm_angle
float32 hand_angle
string  color
int32   count

